docs/wiki/Trade.md

```diff
@@ -0,0 +1,128 @@
+along with access to your trading tier and reputation with the associated DEX server.
+
+
+## Submit Order
+
+The Submit Order section is located at the top of the order panel.
+
+<img src="./images/using-bison-wallet/trade-submitOrder.png" width="1200" alt="">
+
+Read more about order types, how to place an order and trade settlement process
+in the [Order Management](Order-Management) section of this wiki.
+
+## Trading Tier
+
+The trading tier for the associated DCRDEX server account on the selected market can be viewed by
+clicking the `Show Trading Tier Info` button.
+
+<img src="./images/using-bison-wallet/trade-tradingtier-1.png" width="1200" alt="">
+
+
+<img src="./images/using-bison-wallet/trade-tradingtier-2.png" width="400" alt="">
+
+* **Parcel Size:** A server-side configurable parameter on DCRDEX, set individually for each market. 
+It defines the number of [lots](#lot-size) a user can trade based on their trading tier. 
+In markets with low-fee assets, lot sizes may be very small, resulting in a larger parcel size. 
+Conversely, markets with high-fee assets generally have larger lot sizes, where a smaller parcel 
+size may be more appropriate.
+* **Trading Tier:** Reflects your account's current actual trading tier with the DCRDEX server,
+determined based on your active [bonds](./Bison-Wallet-Setup-Guide.md#fidelity-bonds) 
+and [reputation score](#reputation-score).
+When your trading tier falls below 1, your booked orders will be revoked.
+* **Trading Limits:** The maximum number of lots you can trade in the selected market, 
+calculated based on the market’s parcel size and your actual trading tier.
+* **Current Usage:** Shows the proportion of trading limits used. The DCRDEX server tracks usage 
+across all markets, assessing trading limits globally for each account.
+
+## Reputation
+
+The reputation score for the associated DCRDEX server account on the selected market 
+can be viewed by clicking the **Show Reputation** button.
+
+<img src="./images/using-bison-wallet/trade-reputation-1.png" width="1200" alt="">
+
+
+<img src="./images/using-bison-wallet/trade-reputation-2.png" width="400" alt="">
+
+Read more about reputation in the [managing your DCRDEX accounts](#managing-your-dcrdex-accounts)
+section of this wiki.
+
+## Your Orders
+
+This section show your recent orders for the selected market.
+
+<img src="./images/using-bison-wallet/trade-yourOrders-1.png" width="1200" alt="">
+
+Clicking each order will expand it to display additional information. 
+If an order is still in booked status, ti can be cancelled by the 
+<img src="./images/using-bison-wallet/button-cancel.png" width="20" alt=""> button, 
+while the [Order Details](#order-details-view) view can be accessed by clicking
+the <img src="./images/using-bison-wallet/button-link.png" width="20" alt=""> button.
+
+<img src="./images/using-bison-wallet/trade-yourOrders-2.png" width="400" alt="">
```

Comment:
![image](https://github.com/user-attachments/assets/bcce9624-31f6-459a-aff7-217814920dcc)

Order Type has been fixed now and some of the types also have updated translations

![image](https://github.com/user-attachments/assets/df73c7f1-77c1-43e7-9d7e-696765d4ae36)

Reply:
Updated.